### ISSUE TITLE
Reopen: Cannot read properties of undefined (reading '__version')

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#1714]
### Steps to Reproduce

Dear Adam

I'm reopening this issue here because it's not solved yet and I think already closed tickets are filtered-out in your notifications even if i mention you explicitely (using @...).

Here are the details about the still open bug and the details I've added:
https://github.com/ironmansoftware/issues/issues/1659 

Thanks a lot and sorry for bothering...

### Expected behavior

```console
See issue #1659 for details
```

### Actual behavior

```console
See issue #1659 for details
```

### Environment data

3.5.1

**Comment by adamdriscoll** [comment]
@donlent - This is because the tables have the same ID. Using UDHidden is still causing the tables to be registered twice and causing this issue.

**Comment by donlent** [comment]
Dear @adamdriscoll OK thanks. I will re-check everything with the 3.6.x release and give you feedback asap.

**Comment by adamdriscoll** [comment]
@donlent - Do you still have this issue?

**Comment by donlent** [comment]
Hi @adamdriscoll 
You're right I haven't tested yet if it's still an issue. sorry for that. The reason is that I changed job and this project didn't receive the right priority anymore. However today I'll be onsite at the customer and update the dev environment to 3.7.6... WIll let you know immediately after that... :-)
Thanks, Don

**Comment by donlent** [comment]
Sorry Adam today I've been blocked by the customer with many other issues...
So I didn't have the possibility to test... 
Will do it tomorrow (Jan 19th) and will post the results. Thanks for your patience. :-)

**Comment by donlent** [comment]
Hi @adamdriscoll 
As promised I just upgraded from 3.5.4 to 3.7.6 in the test lab.
This particular error message still shows up. As you mentioned earlier it can be avoided by defining different table IDs instead of using the same one. I will let you decide if you want to close this issue or if you want to implement a more "clear" error message if that would happen...
From my side the problem can be considered as resolved, since I've defined all new and separate table IDs...
Thanks, Don
